docs/man/dune.1.md

```diff
@@ -0,0 +1,291 @@
+---
+title: DUNE
+section: 1
+header: User Manual
+footer: DUNE 0.0.1
+date: April 04, 2023
+---
+# NAME
+DUNE - Docker Utilities for Node Execution.
+
+# SYNOPSIS
+
+`dune <option> [<arguments>] [-- <commands>]`
+
+# DESCRIPTION
+
+**DUNE** is a utility to interact with the EOS blockchain system installed in a Docker container. 
+The main purpose of this utility is to provide all features needed for management of EOS in one
+program and make this process simple for non experienced users. DUNE offers functionality to manage
+smart contract development, deployment, and interaction with the blockchain. 
+
+# OPTIONS
+
+**`-- <COMMANDS>`** 
+
+    Adding of double dash and space at the end of the command line allows to call any commands in the container 
+    or append additional options to the end of a current call.
```

Comment:
```suggestion
    Adding a double dash and space at the end of the command line allows for calling any commands in the container
    or appending additional options to the end of a current call.
```

Reply:
in general ENF/EOS docs guidelines avoid gerunds.